musl defines `int ioctl (int, int, ...);` in <sys/ioctl.h>
which is different than glibc and uClib.c which use an
unsigned long int for the second paramater.  `man 2 ioctl`

Since there is no single standard here, meh.

diff -Naur v4l-utils-1.6.2.orig/lib/libv4l1/v4l1compat.c v4l-utils-1.6.2/lib/libv4l1/v4l1compat.c
--- v4l-utils-1.6.2.orig/lib/libv4l1/v4l1compat.c	2015-07-18 14:50:06.220267837 +0000
+++ v4l-utils-1.6.2/lib/libv4l1/v4l1compat.c	2015-07-18 14:51:08.744265485 +0000
@@ -72,7 +72,7 @@
 	return v4l1_dup(fd);
 }
 
-LIBV4L_PUBLIC int ioctl(int fd, unsigned long int request, ...)
+LIBV4L_PUBLIC int ioctl(int fd, int request, ...)
 {
 	void *arg;
 	va_list ap;
diff -Naur v4l-utils-1.6.2.orig/lib/libv4l2/v4l2convert.c v4l-utils-1.6.2/lib/libv4l2/v4l2convert.c
--- v4l-utils-1.6.2.orig/lib/libv4l2/v4l2convert.c	2015-07-18 14:50:06.220267837 +0000
+++ v4l-utils-1.6.2/lib/libv4l2/v4l2convert.c	2015-07-18 14:50:59.348265838 +0000
@@ -96,7 +96,7 @@
 	return v4l2_dup(fd);
 }
 
-LIBV4L_PUBLIC int ioctl(int fd, unsigned long int request, ...)
+LIBV4L_PUBLIC int ioctl(int fd, int request, ...)
 {
 	void *arg;
 	va_list ap;
